specs/experimental/validator-system-v2/asset-manager.md

```diff
@@ -77,113 +104,86 @@ as follows:
 
-## Calculation of Slashing Amount
+## Undelegation Delay of 7 days from the Delegation
 
-If a validator failed to win a challenge, assets of the validator and delegators who delegated their assets to the
-validator are subject to slashing. The slashing amount is determined by the following constants:
+To prevent abuse such as attempting to receive unintended and unjustified rewards or avoiding slashing through various
+forms of manipulation, all undelegation and reward claims are subject to a one-week delay. Without this 7-day delay,
+the following abuses could occur:
 
-| Name                  | Value |
-|-----------------------|-------|
-| `SLASHING_RATE`       | TBD   |
-| `SLASHING_RATE_DENOM` | 1000  |
-| `MIN_SLASHING_AMOUNT` | TBD   |
+> Exploiting the fact that output finalization takes one week, a submitter could delegate a large amount of KRO right
+before output finalization, and then undelegate it immediately after finalization. This way, the delegator could take
+the rewards with almost no risk.
 
-If the slashing amount is too low, the incentive to create challenges would decrease, which could lead to a decrease
-in the network's security. Therefore, `MIN_SLASHING_AMOUNT` is set to ensure the minimum challenge reward.
+To address this edge case, funds are not be able to be undelegated for a week from the time of the delegation. This is
+managed through the `lastDelegatedAt` field in the `KroDelegator` struct and `delegatedAt` mapping `KghDelegator`
+struct respectively.
 
-The slashing amount is calculated as follows:
+## Slashing
 
-$$
-\text{slashingAmount} = \max \left( \frac{(\text{totalAssets} - \text{totalKroInKgh}) \times \text{slashingRate}}
-{\text{slashingRateDenom}}, \text{minSlashingAmount} \right)
-$$
+Since Validator System V2 brings the same bond mechanism of the previous validator system, there is no slashing applied
+to KRO and KGH delegators. They contribute to the security of the chain by delegating to a validator and contributing
+to the probability of the validator to be selected as a priority validator. Since the reward is only given to the
+honest validator, delegators are incentivized to delegate to a validator who is honest and active.
 
-Note that KROs inside KGH are not subject to slashing.
+To see more about the slashing mechanism, please refer to the [Slashing](./validator-manager.md#slashing) part of
+Validator Manager document.
 
 ## Difference from ERC-4626
 
 ### Non-transferable Shares
 
-Unlike ERC-4626, the shares in `Vault` are designed to be non-transferable to any accounts except for the delegator
+Unlike ERC-4626, `kroShares` in `Vault` are designed to be non-transferable to any accounts except for the delegator
 and Asset Manager contract. This is implemented to prevent the following edge cases:
 
 - Transferring shares related to the validator's `minRegisterAmount` (minimum amount of KRO to be a validator) to
```

Comment:
Below sentence should be removed.

`Transferring shares related to KRO within KGH to another address.`

Reply:
Also `However, Vault in Asset Manager manages self-delegation,` to `However, Vault in Asset Manager manages deposit,`